### ISSUE TITLE
BITMAPINFOHEADER ignored

### ISSUE THREAD

**Issue by fcartegnie** [MediaArea/MediaInfoLib#1605]
Tricky issue.

We have received a sample with a strange mix:
* valid BITMAPINFOHEADER RGB header
* mpeg TS as RGB payload
* padding

Mediainfo detects it as MPEG-TS.
To me it's wrong. It shouldn't try to probe further than the bitmap header.

Sample from https://code.videolan.org/videolan/vlc/-/issues/27407

**Comment by JeromeMartinez** [comment]
We definitely have to improve detection by a kind of priority per parser, currently the first parser catching something considered as acceptable "wins".
Note that the same file with ".bmp" extension is correctly analyzed (BMP parser is tested first in that case).

**Comment by JeromeMartinez** [comment]
@fcartegnie I am wondering if it is worth it to detect and report that MPEG-TS content is found inside the fake BMP... Anyway, for when I have spare time for playing with such idiotic file crafted on purpose :-D.

**Comment by robUx4** [comment]
Yes, that file is odd. It has a correct BMP header with correct TS data inside. It's like it was put in a BMP to hide the video.
